lib/mapbox/geocode.py

```diff
@@ -171,6 +171,20 @@ def get_latlng(address, min_relevance_threshold=1):
     return latlng
 
 
+def extract_address_info(response):
+    """Extract address, city, and country_code from a Mapbox feature."""
+    result = {}
+    result['address'] = response[0].get('place_name')
+    for ctx in response[0].get('context', []):
```

Comment:
Is this a list of dicts? Would be helpful to see API docs or sample response in the docstring